appender/pom.xml

```diff
@@ -40,6 +40,19 @@
         <dependency>
             <groupId>redis.clients</groupId>
             <artifactId>jedis</artifactId>
+            <version>${jedis.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.commons</groupId>
+                    <artifactId>commons-pool2</artifactId>
```

Comment:
I had some issues with the default commons-pool2 version coming from redis and overrode it to use a newer one that solved the compilation problems

Reply:
I don't think this is necessary. You have downgraded Jedis from `5.1.2` (which uses `commons-pool:2.12.0` that you are manually trying to inject here) to `5.1.0` in `/pom.xml`. Once you revert all your `pom.xml` changes, I think it will all work.